var/spack/repos/builtin/packages/py-shroud/package.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from spack.hooks.sbang import filter_shebang
+
+
+class PyShroud(Package):
+    """Create Fortran wrappers for a C++ library."""
+
+    homepage = "https://github.com/LLNL/shroud"
+    git      = "https://github.com/LLNL/shroud.git"
+
+    version('develop', branch='develop')
+    version('master',  branch='master')
+    version('0.11.0', tag='v0.11.0')
+    version('0.10.1', tag='v0.10.1')
+    version('0.9.0', tag='v0.9.0')
+    version('0.8.0', tag='v0.8.0')
+
+    extends('python')
+
+    depends_on("py-alabaster")
+    depends_on("py-pytz")
+    depends_on("py-docutils")
+    depends_on("py-setuptools")
+    depends_on("py-pyyaml")
+
+    def install(self, spec, prefix):
+        # simply install to the spack python
+        python('setup.py', 'install')
```

Comment:
We never want to install directly to the Python installation prefix. It makes it impossible to uninstall a package, or install multiple versions of the same package. I would remove this whole `install` method.